README.rst

```diff
@@ -124,10 +124,25 @@ TODO
 * Simple fonctionnal test with real Kinto on TravisCI
 * Handle default bucket
 * Validate API
-* Check record permissions
+
+
+Run tests
+---------
+
+Install `Tox <https://tox.readthedocs.org/>`_ and `Moto <https://github.com/spulec/moto>`_, a fake Amazon S3 server: ::
+
+    pip install tox moto
+
+Run a standalone S3 server in a separate terminal: ::
+
+    moto_server s3bucket_path -H 0.0.0.0 -p 5000
```

Comment:
Then we should probably add a Makefile for ``make install`` and ``make tests``

I filed #20 about that.